contrib/samples/openvino/deployer/deployer.py

```diff
@@ -0,0 +1,32 @@
+import kfp.dsl as dsl
+
+
+@dsl.pipeline(
+    name='OVMS deployment pipeline',
+    description='Deploy OpenVINO Model Server instance in Kubernetes'
+)
+def openvino_predict(
```

Comment:
Can you please also create the `component.yaml` spec file for the component? See https://github.com/kubeflow/pipelines/blob/master/components/sample/keras/train_classifier/component.yaml as an example.

Reply:
The advantage is that the users will be able to just load the component instead of copying and pasting code:
```
openvino_model_server_deployer_op = kfp.components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/contrib/samples/openvino/deployer/component.yaml')
```